src/crimson/osd/osd.cc

```diff
@@ -850,6 +854,18 @@ seastar::future<> OSD::handle_osd_op(ceph::net::Connection* conn,
   return seastar::now();
 }
 
+seastar::future<> OSD::handle_rep_op_reply(ceph::net::Connection* conn,
+					   Ref<MOSDRepOpReply> m)
+{
+  auto pgs = pg_map.get_pgs();
```

Comment:
Is [the copy](https://stackoverflow.com/questions/11886930/auto-specifier-type-deduction-for-references) intended here? Maybe `auto& pgs = ...` would be fine?

Reply:
thanks! i also added `const` to be explicit that we don't change `pgs`.